src/classes/Location.ts

```diff
@@ -0,0 +1,13 @@
+export default class Location {
+    public Id: Number;
+    public Stad: String;
```

Comment:
Change property names to English to match backend model
- City
- Location
- CollectionPoint